module/zfs/arc.c

```diff
@@ -297,6 +297,7 @@
 #include <sys/fs/swapnode.h>
 #include <sys/zpl.h>
 #include <linux/mm_compat.h>
+#include <linux/mm_compat_zfs.h>
```

Comment:
How about we call this `linux/page_compat.h` for now.  It's looking like when the SPL is merged the conflicting `mm_compat.h` can be dropped at which point we can rename this `mm_compat.h` is we like.

Reply:
Yup